spec/index.bs

```diff
@@ -4249,103 +4248,92 @@ The full binding address of a resource can be defined as a trio of:
 
-<dl dfn-type=attribute dfn-for="GPUPipelineLayout">
+<dl dfn-type=attribute dfn-for="pipeline layout">
     : <dfn>\[[bindGroupLayouts]]</dfn> of type [=list=]&lt;{{GPUBindGroupLayout}}&gt;.
     ::
-        The {{GPUBindGroupLayout}} objects provided at creation in {{GPUPipelineLayoutDescriptor/bindGroupLayouts|GPUPipelineLayoutDescriptor.bindGroupLayouts}}.
+        The {{GPUBindGroupLayout}} objects defining the pipeline layout of a pipeline.
+
+        The contents are the same as
+        {{GPUPipelineLayoutRequirements/bindGroupLayouts|GPUPipelineLayoutRequirements.bindGroupLayouts}}
+        passed to pipeline creation, except any {{GPUAutoBindGroupLayoutMode/"auto"}} values
+        are replaced with bind group layouts computed by the [$computed pipeline layout$] algorithm.
 </dl>
 
-Note: using the same {{GPUPipelineLayout}} for many {{GPURenderPipeline}} or {{GPUComputePipeline}} pipelines guarantees that the user agent doesn't need to rebind any resources internally when there is a switch between these pipelines.
+Note: using the same [=pipeline layout=] for many {{GPURenderPipeline}} or {{GPUComputePipeline}} pipelines guarantees that the user agent doesn't need to rebind any resources internally when there is a switch between these pipelines.
 
 <div class="example">
-{{GPUComputePipeline}} object X was created with {{GPUPipelineLayout/[[bindGroupLayouts]]|GPUPipelineLayout.bindGroupLayouts}} A, B, C. {{GPUComputePipeline}} object Y was created with {{GPUPipelineLayout/[[bindGroupLayouts]]|GPUPipelineLayout.bindGroupLayouts}} A, D, C. Supposing the command encoding sequence has two dispatches:
-
-  1. {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|setBindGroup(0, ...)}}
-  1. {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|setBindGroup(1, ...)}}
-  1. {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|setBindGroup(2, ...)}}
-  1. {{GPUComputePassEncoder/setPipeline(pipeline)|setPipeline(X)}}
-  1. {{GPUComputePassEncoder/dispatch(workgroupCountX, workgroupCountY, workgroupCountZ)|dispatch()}}
-  1. {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|setBindGroup(1, ...)}}
-  1. {{GPUComputePassEncoder/setPipeline(pipeline)|setPipeline(Y)}}
-  1. {{GPUComputePassEncoder/dispatch(workgroupCountX, workgroupCountY, workgroupCountZ)|dispatch()}}
-
-In this scenario, the user agent would have to re-bind the group slot 2 for the second dispatch, even though neither the {{GPUBindGroupLayout}} at index 2 of {{GPUPipelineLayout/[[bindGroupLayouts]]|GPUPipelineLayout.bindGrouplayouts}}, or the {{GPUBindGroup}} at slot 2, change.
+    {{GPUComputePipeline}} object X was created with {{pipeline layout/[[bindGroupLayouts]]}} A, B, C. {{GPUComputePipeline}} object Y was created with {{pipeline layout/[[bindGroupLayouts]]}} A, D, C. Supposing the command encoding sequence has two dispatches:
```

Comment:
should "pipeline layout" be a link here?

Reply:
Like:
```suggestion
    {{GPUComputePipeline}} object X was created with a [=pipeline layout=] with {{pipeline layout/[[bindGroupLayouts]]}} A, B, C.

    {{GPUComputePipeline}} object Y was created with a [=pipeline layout=] with {{pipeline layout/[[bindGroupLayouts]]}} A, D, C.

    Supposing the command encoding sequence has two dispatches:
```
?